lib/rsc/story.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import React, { forwardRef } from "react";
+import { useStoryblokState } from "../common/client";
+import { ISbStoryData, StoryblokBridgeConfigV2 } from "../types";
+import { StoryblokComponent } from "./common";
+
+interface StoryblokStoryRSCProps {
+  story: ISbStoryData;
+  bridgeOptions: StoryblokBridgeConfigV2;
+  [key: string]: unknown;
+}
+
+const StoryblokStory = forwardRef<HTMLElement, StoryblokStoryRSCProps>(
+  ({ story, bridgeOptions, ...restProps }, ref) => {
+    if (typeof story.content === "string") {
```

Comment:
Do we care if `story.content` is an empty string? I'm not 100% sure but I think JSON.parse might throw an `SyntaxError: Unexpected end of JSON input`

Reply:
good catch, I will add a try/catch block because for whatever reason it could also be an invalid JSON and it would throw, logging the error and assigning an empty object to content here